functions.php

```diff
@@ -66,15 +71,19 @@ function judge_familiar_page_navigation_box( $page ) {
 
 		$prev = get_post_meta( $page->ID, '_previous_page', true );
 		$next = get_post_meta( $page->ID, '_next_page', true );
+		$prev_title = get_post_meta( $page->ID, '_previous_page_title', true );
+		$next_title = get_post_meta( $page->ID, '_next_page_title', true );
 ?>
 		<div class="page_navigation_input">
 			<label for="page_navigation_input_prev">Previous Page</label>
-			<input id="page_navigation_input_prev" name="page_navigation_input_prev" type="text" value="<?= $prev ?>" />
+			<input id="page_navigation_input_prev" name="page_navigation_input_prev" type="number" value="<?= $prev ?>" placeholder="Page ID"/><!--
+			--><input id="page_navigation_input_prev_title" name="page_navigation_input_prev_title" type="text" value="<?= $prev_title ?>" placeholder="Page Title" />
```

Comment:
Yes, the comment is necessary. If it were a space / line break instead, you'd break the 2 input fields into 2 lines. So it's either a comment (for readability) or the second input has to be directly next to the first one without a space.